astroquery/skyview/tests/test_skyview_remote.py

```diff
@@ -15,11 +16,9 @@ def test_get_image_list():
 
 @remote_data
```

Comment:
How big are the files?  If they're more than a few MB, could you add a second decorator here, `@pytest.mark.bigdata`?